webpack.config.js

```diff
@@ -12,8 +12,9 @@ module.exports = {
     path: path.join(__dirname, 'dist'),
     filename: '[name].js'
   },
+  mode: 'development',
```

Comment:
I guess it should depend on `PRODUCTION` env variable (that is set by package.json script).